app/routes/bill-runs.routes.js

```diff
@@ -18,6 +18,23 @@ const routes = [
       },
       description: 'Used to create a bill run'
     }
+  },
+
+  {
+    method: 'GET',
+    path: '/bill-runs/{id}/review',
+    handler: BillRunsController.review,
+    options: {
+      app: {
+        plainOutput: true
+      },
+      auth: {
+        access: {
+          scope: ['billing']
+        }
+      },
+      description: 'Used to review sroc returns data'
```

Comment:
I might suggest this as an alternate description. 🤷 

```suggestion
      description: 'Review two-part tariff match and allocation results'
```